### PR TITLE
Add support for negative numbers to isFloat

### PR DESCRIPTION
**Pull request by kizaonline**

### CODE REVIEW

File: src/ValidationRules.jsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ const validations = {
 
     isNumber: value => validations.matchRegexp(value, /^-?[0-9]\d*(\d+)?$/i),
 
-    isFloat: value => validations.matchRegexp(value, /^(?:[1-9]\d*|0)?(?:\.\d+)?$/i),
+    isFloat: value => validations.matchRegexp(value, /^(?:-?[1-9]\d*|-?0)?(?:\.\d+)?$/i),
```
Comment (NewOldMax) [inline src/ValidationRules.jsx:36]: first `-?` is ok, byt why is second one? to handle `-0`? does it make sense?

Comment (kizaonline) [reply src/ValidationRules.jsx:36]: Hi @NewOldMax, yes to handle -0, theoretically valid but happy to remove it from a practical perspective

**Comment by NewOldMax** [comment]
Also add new rule to rule list in readme

**Comment by kizaonline** [comment]
This is an update to an existing rule, it is already listed in the readme, do you want something more descriptive added to cover that `isFloat` now allows negatives?

File: src/ValidationRules.jsx (hunk 1 above)
Comment (kizaonline) [reply src/ValidationRules.jsx:36]: I've added the -0 back in, to allow for -0.X floats

**Comment by NewOldMax** [comment]
thanks for PR

**Pull request merged**